src/applications/caregivers/config/chapters/signAsRepresentative/signAsRepresentativeYesNo.js

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+// import fullSchema from 'vets-json-schema/dist/10-10CG-schema.json';
+import { representativeFields } from 'applications/caregivers/definitions/constants';
+import AdditionalInfo from '@department-of-veterans-affairs/component-library/AdditionalInfo';
+
+// const { representative } = fullSchema.properties;
+// const veteranProps = veteran.properties;
+// const { address, phone } = fullSchema.definitions;
+
+const RepresentativeIntroContent = () => {
+  return (
+    <section>
+      <p>
+        Some family caregivers are also the Veteran’s legal representative.
+        These representatives have the legal authority to make certain decisions
+        for the Veteran.
+      </p>
+
+      <p>Here's what you should know</p>
+      <ul>
+        <li>
+          You can still continue with this application to apply for the program
+          even if you’re not the Veteran’s legal representative.
+        </li>
+
+        <li>
+          If you are the Veteran’s legal representative, you can upload one or
+          more documents to show your legal status. If you don't upload your
+          documents now, we'll ask you to provide them later
+        </li>
+      </ul>
+    </section>
+  );
+};
+
+const RepresentativeAdditionalInfo = () => {
+  return (
+    <div className="vads-u-margin-top--1">
+      <AdditionalInfo triggerText="What documents can I submit to show legal status as a representative?">
+        <ul>
+          <p>
+            Documentation to show your legal status as a representative could
+            include:
+          </p>
```

Comment:
A `<p>` shouldn't be a child of a `<ul>`

Reply:
Yeah I was thinking about that I moved it out due to a indent but I think you are right just need to remove that indent

Reply:
100% good catch!